### ISSUE TITLE
Unable to create docs with custom type arrays.

### ISSUE THREAD

**Issue by meizon** [bijoutrouvaille/fireward#18]
If I create the following:

```
type Session = {
  dayOfWeek: int
}

type Booking = {
  sessions = [Session, Session?]
}
```

I am not able to create new `Booking` objects with 1 or more `Session` objects, however if I create an object in the firestore console, I can then _update_ that `Booking` object and change the values of the `dayOfWeek` value and they work fine.

**Comment by meizon** [comment]
After trying this again, it seems that I cannot edit existing `Booking` objects.

**Comment by bijoutrouvaille** [comment]
@meizon thank you for posting the bug. It should now be fixed in version 1.4.2 on NPM and other channels.
